org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -48,6 +48,7 @@ public void testCoverageResult() {
 		// 3. Try/Catch Block Without Exception Thrown
 		assertLine("noExceptionTryCatch.beforeBlock", ICounter.FULLY_COVERED);
 		assertLine("noExceptionTryCatch.tryBlock", ICounter.FULLY_COVERED);
+		assertLine("noExceptionTryCatch.catch", ICounter.NOT_COVERED);
```

Comment:
Might just remove marker, however knowing that there is difference in bytecode generated by ECJ and without looking into it easy to expect "empty" or "not covered", but wondering why "partly"? :confused: